app/Http/Controllers/Api/V1/Integration/BrightcoveAPISettingsController.php

```diff
@@ -96,10 +97,14 @@ public function store(StoreBrightcoveAPISetting $request, Organization $suborgan
     }
 
     /**
+     * Update Brightcove API Setting
+     * 
      * Update Brightcove API Setting Data
-     * @param id required The Id of a brightcove_api_settings table Example: 1
+     * 
      * @param UpdateBrightcoveAPISetting $request
```

Comment:
Replace "@param" with "@urlParam"  as mentioned in 
https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api